frontend/src/components/common/header/common/hooks/use-visible.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
```

Comment:
Move it to the `frontend/src/hooks`
I think that we will reuse it in the future